### DISCUSSION TITLE
Down to basic for threejs simple set up. What the correct way to set up?

### DISCUSSION THREAD

**Discussion by Lightnet** [jrouwe/JoltPhysics.js#221] [Q&A]
I try to search engine. Although I did look at the examples but I can't understand well. I mean it different program language docs. I don't see javascript docs for correct way to set up just ground and cube drop test. There one example but not much still got some "[HelloWorld.js](https://github.com/jrouwe/JoltPhysics.js/blob/main/helloworld/HelloWorld.js)" refs.

For example that rapierjs has some clear docs with proper set up in javascript. https://rapier.rs/docs/user_guides/javascript/getting_started_js

Wanted to see strip down version. Meaning just threejs and jolt with out the helper script. I did see helper script but I lack how it work correctly. Although I manage to get simple debug test on local build.

https://gist.github.com/Lightnet/28e0c92a930a379125c9e8b9996f1148

I see five step to make box added to physics world objects. Well trying to how set up works. I hope explain works. Tying to create docs to help me remember how to set up the shapes. Did strip down bit base on the example files. Not sure correct way set up.

**Comment by LeXXik** [comment]
Not sure what to recommend here, but the examples are straight forward and easy to follow. 

In any case, you should read the documentation first. Even though it is for C++ version, it is still relevant and explains how to work with Jolt. After that you can go through examples to see how they are done.

https://jrouwe.github.io/JoltPhysics/index.html#architecture-jolt-physics

https://github.com/jrouwe/JoltPhysics.js/tree/main/Examples

**Comment by Lightnet** [comment]
Well c++ docs. I do not understand a bit. Although it quick look. I guess I can refs them. But it need proper docs examples. It very strange but guessing pro coder but not beginner coder examples. It will take a while to understand. It not easy to translate as no pro c++ understand of it. What I is in javascript docs. Well I have try and going to waste hours to understand the code. Knowing there more features to translate and understand beyond like clothing physics.
